www/cbbs_imaging/index.html

```diff
@@ -0,0 +1,510 @@
+      <p>Which data analysis software do you use?</p>
+      <div class='checkboxes extendable'>
+        <label><input type='checkbox' name='sw_list[]'
+          value='AFNI'> AFNI</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='brainvisa'> brainvisa</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='BrainVoyager'> BrainVoyager</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='Camino'> Camino</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='DTI-Query'> DTI-Query</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='DTIStudio'> DTIStudio</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='DTItk'> DTItk</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='Freesurfer'> Freesurfer</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='FSL'> FSL</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='ImageJ'> ImageJ</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='ITK'> ITK</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='Lipsia'> Lipsia</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='LORIS'> LORIS</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='mipav'> mipav</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='mrtrix'> mrtrix</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='nibabel'> nibabel</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='nipy'> nipy</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='PROBID'> PROBID</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='PyMVPA'> PyMVPA</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='scikit-learn'> scikit-learn</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='SPM'> SPM</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='vistasoft'> vistasoft</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='Trackvis'> Trackvis</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='Voxbo'> Voxbo</label>
+        <label><input type='checkbox' name='sw_list[]'
+          value='VTK'> VTK</label>
+      </div>
+    </div>
+
+    <div class="tl-panel badge-heart">
+      <h2>Desired analyses</h2>
+      <p>What analysis procedures/workflows would you most appreciate us to make available?</p>
+      <input name="analyses_desired" type="text">
+    </div>
+
+    <div class="tl-panel badge-upload">
+      <h2>Your contribution</h2>
+      <p>What methods/tools developed in your lab would you like to become available?</p>
```

Comment:
For others, for self? What's primary motivation? I think people will parse this statement.